regression-test/suites/query_p0/system/test_storage_page_size.groovy

```diff
@@ -0,0 +1,63 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import org.codehaus.groovy.runtime.IOGroovyMethods
+
+suite ("test_storage_page_size") {
+
+    sql """ DROP TABLE IF EXISTS table_1; """
+
+    sql """
+        create table table_1(
+            k1 int not null,
+            k2 int not null,
+            k3 bigint null,
+            k4 varchar(100) null
+        )
+        duplicate key (k1)
+        distributed BY random buckets 1
+        properties("replication_num" = "1");
+    """
+
+    test {
+        sql "show create table table_1;"
+        check { result, exception, startTime, endTime ->
+            assertTrue(result[0][1].contains("\"storage_page_size\" = \"65536\""))
+        }
+    }
+
+    sql """ DROP TABLE IF EXISTS table_2; """
+
+    sql """
+        create table table_2 (
```

Comment:
add test for more values like <<min, min-1, min, min+1, min~max, max-1,max,max+1, >>max